packages/replay/metadata/test.ts

```diff
@@ -13,30 +15,30 @@ const isUuid = require("is-uuid");
 
 const VERSION = 1;
 
-const versions = {
+const versions: Record<number, Struct> = {
   1: object({
     file: optional(string()),
     path: optional(array(string())),
     result: enums(["passed", "failed", "timedOut"]),
     run: optional(object({
-      id: define('uuid', (v) => isUuid.v4(v)),
+      id: define('uuid', (v: any) => isUuid.v4(v)),
       title: optional(string())
     })),
     title: string(),
     version: defaulted(number(), () => 1),
   }),
 };
 
-function validate(metadata) {
+function validate(metadata: {test: Record<string, unknown>}) {
   if (!metadata || !metadata.test) {
     throw new Error('Test metadata does not exist');
   }
 
   return init(metadata.test);
 }
 
-function init(data) {
-  const version = data.version || VERSION;
+function init(data: Record<string, unknown>) {
```

Comment:
:hand: given that we have `Record<string, unknown>` popping up quite a bit, I'll often extract that into its own type just so I don't repeat myself